.github/workflows/r-package-ubsan.yml

```diff
@@ -0,0 +1,32 @@
+name: R package (ASAN/UBSAN)
+
+on:
+  pull_request_review_comment:
+    types: [created]
+
+jobs:
+  test:
+    name: Debian Linux, R-devel, GCC ASAN/UBSAN
+    if: github.event.comment.body == '/gha run r-sanitizers-check' && contains('OWNER,MEMBER,COLLABORATOR', github.event.comment.author_association)
```

Comment:
/gha run r-sanitizers-check

Reply:
/gha run r-sanitizers-check

Reply:
^ this comment triggered https://github.com/microsoft/LightGBM/pull/3439/checks?check_run_id=1212762820

The first comment didn't work because it was a normal comment, not a "review comment"

Reply:
/gha run r-sanitizers-check